examples/validate-fee/sample-ui/src/app/polls/create/actions.ts

```diff
@@ -0,0 +1,63 @@
+'use server';
+
+import { revalidatePath, revalidateTag } from 'next/cache';
+import { redirect } from 'next/navigation';
+import { z } from 'zod';
+
+import { PollSchema } from '../../../schemas';
+import { MetagraphBaseURLs } from '../../../consts';
+
+const CreatePollSchema = PollSchema.extend({
+  signedPayload: z
+    .string({ invalid_type_error: 'Invalid signed payload' })
+    .regex(
+      /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/,
+      'Invalid base64 character'
+    )
```

Comment:
I see this regex third or fourth time. Maybe it's worth to put it in one place and reuse?

Reply:
Deleted sample-ui